src/Moryx.Resources.Benchmarking/AssemblyCell.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) 2023, Phoenix Contact GmbH & Co. KG
+// Licensed under the Apache License, Version 2.0
+
+using System.Collections.Generic;
+using Moryx.AbstractionLayer.Drivers.Message;
+using Moryx.AbstractionLayer.Resources;
+using Moryx.ControlSystem.Activities;
+using Moryx.ControlSystem.Cells;
+using Moryx.Resources.Benchmarking.Messages;
+
+namespace Moryx.Resources.Benchmarking
```

Comment:
Did you add this from scratch? Maybe you could extend the PR message just a bit, cause I don't know what to make of this addition 🙈

Reply:
No, I moved it from the simulation samples. They must have gotten restored during a merge. I cleaned it up and also renamed the module to match the other ControlSystem modules.